tests/spec/core/data-abbr-spec.js

```diff
@@ -1,57 +1,48 @@
 "use strict";
-describe("Core — Definition Abbreviations", () => {
+describe("Core — data-abbr", () => {
   afterAll(flushIframes);
   it("processes definition abbreviations", async () => {
     const ops = {
       config: makeBasicConfig(),
       body: `
       <section id="section">
-        <dfn data-abbr>foo bar</dfn> can be referenced as
-        <span id="foobarLink">
-          <a id="dfnlink">FB</a>
-        </span>
+        <dfn data-abbr>foo bar</dfn>
       </section>`,
     };
     const doc = await makeRSDoc(ops);
-
     const dfnFooBar = doc.querySelector("#section dfn");
-    expect(dfnFooBar.id).toBe("dfn-fb");
+    const abbr = doc.querySelector("#section dfn + abbr");
+
     expect(dfnFooBar.dataset.abbr).toBe("FB");
-    expect(dfnFooBar.textContent.trim()).toBe("foo bar (FB)");
-    const linkToDfn = doc.getElementById("dfnlink");
-    expect(linkToDfn.getAttribute("href")).toBe("#dfn-fb");
-    expect(linkToDfn.querySelector("abbr").textContent.trim()).toBe("FB");
+    expect(dfnFooBar.dataset.lt).toBe("fb|foo bar");
+    expect(dfnFooBar.textContent.trim()).toBe("foo bar");
+    expect(abbr.textContent).toBe("FB");
+    expect(abbr.previousSibling.textContent.trim()).toBe("(");
+    expect(abbr.nextSibling.textContent.trim()).toBe(")");
+
+    expect(abbr.title).toBe("foo bar");
   });
   it("allows different abbreviation combinations to link to dfn", async () => {
     const ops = {
       config: makeBasicConfig(),
       body: `
       <section id="section">
         <dfn data-abbr>foo bar</dfn> can be referenced as
-        <span id="foobarLink">
-          <a class="dfnlink">FB</a>
-          <a class="dfnlink">foo bar</a>
-          <a class="dfnlink">foo bar (FB)</a>
-        </span>
+        <a class="dfnlink">FB</a>
+        <a class="dfnlink">foo bar</a>
       </section>`,
     };
     const doc = await makeRSDoc(ops);
-
-    const dfnFooBar = doc.querySelector("#section dfn");
-    expect(dfnFooBar.id).toBe("dfn-fb");
-    expect(dfnFooBar.dataset.abbr).toBe("FB");
-    expect(dfnFooBar.textContent.trim()).toBe("foo bar (FB)");
-    const linkToDfn = doc.querySelectorAll(".dfnlink");
-    linkToDfn.forEach(link =>
-      expect(link.getAttribute("href")).toBe("#dfn-fb")
-    );
+    doc
+      .querySelectorAll(".dfnlink")
+      .forEach(link => expect(link.getAttribute("href")).toBe("#dfn-fb"));
   });
   it("correctly parses and abbreviates different Abbreviation syntax", async () => {
     const ops = {
       config: makeBasicConfig(),
       body: `
       <section id="section">
-        <dfn data-abbr>United States</dfn>
+        <dfn data-abbr="">United States</dfn>
```

Comment:
Is there a difference? The issue originally asked for the the former way. (```<dfn data-abbr>```)

Reply:
We're testing `<dfn data-abbr>` above already :)

Reply:
oh right.